abis/DutchAuctionRebalancer.ts

```diff
@@ -321,5 +321,25 @@ export const DutchAuctionRebalancerAbi = [
         "type": "error",
         "name": "Unauthorized",
         "inputs": []
+    },
+] as const;
+
+export const DutchAuctionRebalancerAbi = [
+    ...DutchAuctionRebalancerBaseAbi,
+    {
+        "type": "error",
+        "name": "AuctionNotValid",
+        "inputs": []
+    },
+    {
+        "type": "error",
+        "name": "InvalidLeverageTokenStateAfterRebalance",
+        "inputs": [
+            {
+                "name": "leverageToken",
+                "type": "address",
+                "internalType": "address"
+            },
+        ]
```

Comment:
also fixed this so that on tx revert for taking auction we can parse the error (these errors are emitted by the RebalanceAdapter, but if we dont include here viem will not parse it and instead show an encoded error, in `auctionCreated` we handle the parsed errors so we can log them better). I had included these in this abi in the past but mustve removed them by accident when updating this abi at some point. Now using this pattern which will avoid making that same mistake

Reply:
Nice catch. Do we want to manually add these errors to ABIs or do something like .filter/.map the dependent ABIs to add all custom errors in a dynamic way?

Reply:
hmmm can you give an example / pseudocode of what you mean exactly? do you mean importing abis from another file e.g. an `abis/index.ts` that imports and exports all of the abis, and also adds the errors in there? So we would only import abis from this file?